app/models/skin_parent.rb

```diff
@@ -1,6 +1,6 @@
 class SkinParent < ApplicationRecord
-  belongs_to :child_skin, class_name: "Skin", inverse_of: :skin_parents, touch: true
-  belongs_to :parent_skin, class_name: "Skin", inverse_of: :skin_children
+  belongs_to :child_skin, class_name: "Skin", touch: true
+  belongs_to :parent_skin, class_name: "Skin"
```

Comment:
There is a comment about why `inverse_of:` is fine to remove on [creatorship.rb](https://github.com/otwcode/otwarchive/pull/4690/files#diff-174381727fd93f2148df7f93d54cad663246c8bc9fa72b3a31c258c76861de63).

However, I think it's not fine in this case. I'm re-reading the rails guide on associations for another PR and found [this](https://guides.rubyonrails.org/v6.1/association_basics.html#bi-directional-associations):
> However, Active Record will not automatically identify bi-directional associations that contain a scope or any of the following options:
>     :through
>     :foreign_key

Both `skin_parents` and `skin_children` use `:foreign_key`, so I think this removal breaks the bi-directionality.

Reply:
I think we're OK in this case, since `Skin` already has `inverse_of` as well: https://github.com/otwcode/otwarchive/blob/54e5f2640a129acd3a1e593362c62bc0a9b01957/app/models/skin.rb#L40-L47 I tried the experiment listed in the docs, and got the second (expected) result without `inverse_of` in `SkinParent`:

```
irb(main):001:0> skin = Skin.second
  Skin Load (0.5ms)  SELECT `skins`.* FROM `skins` ORDER BY `skins`.`id` ASC LIMIT 1 OFFSET 1
=> 
#<Skin:0x0000558c0be25210
...
irb(main):002:0> skin_parent = skin.skin_children.first
  SkinParent Load (0.3ms)  SELECT `skin_parents`.* FROM `skin_parents` WHERE `skin_parents`.`parent_skin_id` = 2 ORDER BY `skin_parents`.`id` ASC LIMIT 1
=> 
#<SkinParent:0x0000558c0bd6a028
...
irb(main):003:0> skin.title == skin_parent.parent_skin.title
=> true
irb(main):004:0> skin.title
=> "Archive 2.0: (1) core"
irb(main):005:0> skin.title = 'Foobar'
=> "Foobar"
irb(main):006:0> skin.title == skin_parent.parent_skin.title
=> true
```

Reply:
Interesting to see the test of the example, thank you! Looks indeed fine then.

For some further reading, I found this discussion which argues that `inverse_of` on the `belongs_to` side of the `has_many` is never really helpful: https://github.com/rubocop/rubocop-rails/issues/632 (note that it is outdated regarding has_many_inversing).